{
  "dataType": "CVE_RECORD",
  "containers": {
    "adp": [
      {
        "title": "CVE Program Container",
        "references": [
          {
            "url": "https://www.ibm.com/support/pages/node/7156941",
            "tags": [
              "vendor-advisory",
              "x_transferred"
            ]
          },
          {
            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/283364",
            "tags": [
              "vdb-entry",
              "x_transferred"
            ]
          }
        ],
        "providerMetadata": {
          "orgId": "af854a3a-2127-422b-91ae-364da2661108",
          "shortName": "CVE",
          "dateUpdated": "2024-08-01T23:36:21.373Z"
        }
      },
      {
        "title": "CISA ADP Vulnrichment",
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2024-25053",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "partial"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-07-01T20:57:31.396477Z"
              }
            }
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-07-01T20:57:34.844Z"
        }
      }
    ],
    "cna": {
      "title": "IBM Cognos Analytics improper certificate validation",
      "source": {
        "discovery": "UNKNOWN"
      },
      "metrics": [
        {
          "format": "CVSS",
          "cvssV3_1": {
            "scope": "UNCHANGED",
            "version": "3.1",
            "baseScore": 5.9,
            "attackVector": "NETWORK",
            "baseSeverity": "MEDIUM",
            "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:H/A:N",
            "integrityImpact": "HIGH",
            "userInteraction": "NONE",
            "attackComplexity": "HIGH",
            "availabilityImpact": "NONE",
            "privilegesRequired": "NONE",
            "confidentialityImpact": "NONE"
          },
          "scenarios": [
            {
              "lang": "en",
              "value": "GENERAL"
            }
          ]
        }
      ],
      "affected": [
        {
          "cpes": [
            "cpe:2.3:a:ibm:cognos_analytics:11.2.0:*:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:cognos_analytics:11.2.1:*:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:cognos_analytics:11.2.2:*:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:cognos_analytics:11.2.3:*:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:cognos_analytics:11.2.4:*:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:cognos_analytics:12.0.0:*:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:cognos_analytics:12.0.1:*:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:cognos_analytics:12.0.2:*:*:*:*:*:*:*"
          ],
          "vendor": "IBM",
          "product": "Cognos Analytics",
          "versions": [
            {
              "status": "affected",
              "version": "11.2.0, 11.2.1, 11.2.2, 11.2.3, 11.2.4, 12.0.0, 12.0.1, 12.0.2"
            }
          ],
          "defaultStatus": "unaffected"
        }
      ],
      "references": [
        {
          "url": "https://www.ibm.com/support/pages/node/7156941",
          "tags": [
            "vendor-advisory"
          ]
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/283364",
          "tags": [
            "vdb-entry"
          ]
        }
      ],
      "x_generator": {
        "engine": "Vulnogram 0.2.0"
      },
      "descriptions": [
        {
          "lang": "en",
          "value": "IBM Cognos Analytics 11.2.0, 11.2.1, 11.2.2, 11.2.3, 11.2.4, 12.0.0, 12.0.1, and 12.0.2 is vulnerable to improper certificate validation when using the IBM Planning Analytics Data Source Connection. This could allow an attacker to spoof a trusted entity by interfering in the communication path between IBM Planning Analytics server and IBM Cognos Analytics server.  IBM X-Force ID:  283364.",
          "supportingMedia": [
            {
              "type": "text/html",
              "value": "IBM Cognos Analytics 11.2.0, 11.2.1, 11.2.2, 11.2.3, 11.2.4, 12.0.0, 12.0.1, and 12.0.2 is vulnerable to improper certificate validation when using the IBM Planning Analytics Data Source Connection. This could allow an attacker to spoof a trusted entity by interfering in the communication path between IBM Planning Analytics server and IBM Cognos Analytics server.  IBM X-Force ID:  283364.",
              "base64": false
            }
          ]
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "CWE",
              "cweId": "CWE-295",
              "description": "CWE-295 Improper Certificate Validation"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "shortName": "ibm",
        "dateUpdated": "2024-06-28T18:53:48.828Z"
      }
    }
  },
  "cveMetadata": {
    "cveId": "CVE-2024-25053",
    "state": "PUBLISHED",
    "dateUpdated": "2024-08-01T23:36:21.373Z",
    "dateReserved": "2024-02-03T14:49:42.764Z",
    "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
    "datePublished": "2024-06-28T18:53:48.828Z",
    "assignerShortName": "ibm"
  },
  "dataVersion": "5.1"
}